.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   ci-format:
     strategy:
       matrix:
-        os: [ubuntu-20.04]
+        os: [ubuntu-22.04]
```

Comment:
Should we split this PR up more? A few of these changes could just go right in which would reduce the diff and make reverting any regressions easier